bin/check-ssl-qualys.rb

```diff
@@ -107,11 +129,37 @@ def ssl_check(from_cache)
 
   def ssl_recheck
     1.upto(config[:num_checks]) do |step|
-      json = ssl_check(step != 1)
+      p "step: #{step}" if config[:debug]
+      start_time = Time.now
+      p "start_time: #{start_time}" if config[:debug]
+      json = if step == 1
```

Comment:
Couldn't this be shorted to be `ssl_check(step == 1)`?

Reply:
It could and is closer to what was originally there. I write my code for 3 AM troubleshooting as that is when I care the most about being able to grok it. It's more so a matter of preference.